[2024-09-22T15:16:36.428+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-22T15:16:36.528+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrap_carmudi.modify_data scheduled__2024-09-07T00:00:00+00:00 [queued]>
[2024-09-22T15:16:36.535+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrap_carmudi.modify_data scheduled__2024-09-07T00:00:00+00:00 [queued]>
[2024-09-22T15:16:36.536+0700] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-09-22T15:16:36.547+0700] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): modify_data> on 2024-09-07 00:00:00+00:00
[2024-09-22T15:16:36.550+0700] {standard_task_runner.py:64} INFO - Started process 243977 to run task
[2024-09-22T15:16:36.552+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'scrap_carmudi', 'modify_data', 'scheduled__2024-09-07T00:00:00+00:00', '--job-id', '256', '--raw', '--subdir', 'DAGS_FOLDER/carmudi/scrape_carmudi.py', '--cfg-path', '/tmp/tmph_puwror']
[2024-09-22T15:16:36.554+0700] {standard_task_runner.py:91} INFO - Job 256: Subtask modify_data
[2024-09-22T15:16:36.603+0700] {task_command.py:426} INFO - Running <TaskInstance: scrap_carmudi.modify_data scheduled__2024-09-07T00:00:00+00:00 [running]> on host BrightHorizons.
[2024-09-22T15:16:36.689+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrap_carmudi' AIRFLOW_CTX_TASK_ID='modify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-07T00:00:00+00:00'
[2024-09-22T15:16:36.689+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-22T15:16:36.708+0700] {base.py:84} INFO - Using connection ID 'spark_local' for task execution.
[2024-09-22T15:16:36.709+0700] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master local[*] --name arrow-spark --deploy-mode client /home/miracle/mobil/spark/carmudi.py
[2024-09-22T15:16:38.399+0700] {spark_submit.py:579} INFO - 24/09/22 15:16:38 WARN Utils: Your hostname, BrightHorizons resolves to a loopback address: 127.0.1.1; using 10.255.255.254 instead (on interface lo)
[2024-09-22T15:16:38.402+0700] {spark_submit.py:579} INFO - 24/09/22 15:16:38 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-09-22T15:16:38.951+0700] {spark_submit.py:579} INFO - python3: can't open file '/home/miracle/mobil/spark/carmudi.py': [Errno 2] No such file or directory
[2024-09-22T15:16:38.968+0700] {spark_submit.py:579} INFO - 24/09/22 15:16:38 INFO ShutdownHookManager: Shutdown hook called
[2024-09-22T15:16:38.969+0700] {spark_submit.py:579} INFO - 24/09/22 15:16:38 INFO ShutdownHookManager: Deleting directory /tmp/spark-ea109d6a-f4aa-44da-adae-e3992c7f23bb
[2024-09-22T15:16:38.993+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-22T15:16:39.004+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master local[*] --name arrow-spark --deploy-mode client /home/miracle/mobil/spark/carmudi.py. Error code is: 2.
[2024-09-22T15:16:39.007+0700] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=scrap_carmudi, task_id=modify_data, run_id=scheduled__2024-09-07T00:00:00+00:00, execution_date=20240907T000000, start_date=20240922T081636, end_date=20240922T081639
[2024-09-22T15:16:39.017+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 256 for task modify_data (Cannot execute: spark-submit --master local[*] --name arrow-spark --deploy-mode client /home/miracle/mobil/spark/carmudi.py. Error code is: 2.; 243977)
[2024-09-22T15:16:39.055+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-09-22T15:16:39.073+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-22T15:16:39.074+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
